bode/bode/models/task.py

```diff
@@ -31,7 +28,7 @@ class Task(db.Model):
     title = db.Column(db.String(80), nullable=False)
     description = db.Column(db.String(1024), nullable=False, server_default="")
     due_date = db.Column(UTCDateTime(), nullable=True)
-    status = db.Column(Enum(TaskStatus), nullable=False, server_default="TODO")
+    status = db.Column(Enum(*TaskStatus.list(), name="task_status"), nullable=False, server_default="TODO")
```

Comment:
There's a change in a model but no migration. Not sure if it is needed but the enum name in the last migration was `"taskstatus"`